src/overlays/actors/ovl_En_Mu/z_en_mu.h

```diff
@@ -6,9 +6,17 @@
 
 struct EnMu;
 
+typedef void (*EnMuActionFunc)(struct EnMu*, struct GlobalContext*);
+
 typedef struct EnMu {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x100];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ EnMuActionFunc actionFunc;
+    /* 0x0194 */ ColliderCylinder collider;
+    /* 0x01E0 */ char dialogState[0x28];
+    /* 0x0208 */ u16 defFaceReaction;
+    /* 0x020A */ s16 jointTable[0x10];
+    /* 0x022A */ s16 morphTable[0x11];
```

Comment:
```suggestion
    /* 0x020A */ s16 jointTable[16];
    /* 0x022A */ s16 morphTable[17];
```

Reply:
morphTable is also size 16. But more pressingly, neither of these is the morph or joint table, which are dynamically allocated and not part of the actor struct. I'm not entirely sure what they are, but they have to do with whatever `func_80034F54` is doing. These probably need to go back to `unk_20A` and `unk_22A`.

Reply:
Okay no problem. I could see why the size should match.

I selected those variable names given how close they were to 'z_en_go.c's use of `func_80034F54` but I did notice many actors did not name them so it may be best to leave it.